src/style.css

```diff
@@ -1,3 +1,79 @@
+html,
 body {
+  padding: 0;
   margin: 0;
+  width: 100%;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't work. It's a good practice to specify a generic family at the end of the list as a last resort. For example: `font-family: Roboto, Arial, sans-serif;`.